lib/screens/changelog_screen.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:developer';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+// TODO Check if the app was updated -> If yes, show this page, make accessible from settings?
```

Comment:
i think making this show again by pressing something in the settings or about screen is good idea.

Reply:
@timosturm Did you see this? Do you want to do this in the PR?

Reply:
@nilsbehlen @cornelinux I would like to add this to the about screen, but that would have to be designed first. I think this is out of the scope of this pull request. I already worked on that screen, but I will do this in another pull request.